.github/workflows/build.yml

```diff
@@ -92,4 +107,20 @@ jobs:
           context: .
           file: docker/Dockerfile.linux
           push: false
-          tags: chocolatey/choco:latest
\ No newline at end of file
+          tags: chocolatey/choco:latest
+          build-args: |
+            "github_actions=true"
+            "github_ref=${{ github.ref }}"
+            "github_repository=${{ github.repository }}"
+            "github_base_ref=${{ github.base_ref }}"
+            "github_head_ref=${{ github.head_ref }}"
+            "github_run_number=${{ github.run_number }}"
+  finish:
```

Comment:
Maybe rename this to something a bit more specific like `finish-coverage-upload`?

Reply:
Yes, I think that this is a good idea!